src/triton_fil/triton_tensor.cuh

```diff
@@ -84,6 +84,7 @@ class TritonTensor {
       stream_{stream},
       buffer{[&] {
         if (is_owner_) {
+          LOG_MESSAGE(TRITONSERVER_LOG_INFO, "Copying to device??");
```

Comment:
I see this message being printed, indicating that the data somehow gets copied over to the GPU memory. Trying to figure out how to pin the data to the host memory.